controllers/ptpoperatorconfig_controller.go

```diff
@@ -214,7 +214,15 @@ func (r *PtpOperatorConfigReconciler) syncLinuxptpDaemon(ctx context.Context, de
 		}
 	}
 
-	enabledPlugins := strings.Join(defaultCfg.Spec.EnabledPlugins, ",")
+	var pluginList []string
+
+	if defaultCfg.Spec.EnabledPlugins != nil {
+		pluginList = *defaultCfg.Spec.EnabledPlugins
+	} else {
+		pluginList = []string{"e810"} // Enable e810 by default if plugins not specified
+	}
+
+	enabledPlugins := strings.Join(pluginList, ",")
 	data.Data["EnabledPlugins"] = enabledPlugins
 	if enabledPlugins != "" {
```

Comment:
So the logic is you control plugins enable and disable options from ptpoperator ,correct ?

Reply:
This would set the environment variable the daemon uses to read which plugins to register, and then the plugin will act based on the ptpconfig.
We can revisit this in the future when we are working on hwconfig population from the plugin.

Reply:
understood